.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: java
+before_install:
```

Comment:
Why use before_install instead of just making this install? Because the install phase is meant to setup all the dependencies which is what this seems to be doing.

Reply:
done